tools/scripts/scvi/scvi-train.py

```diff
@@ -0,0 +1,61 @@
+import cellxgene_census
+import tiledbsoma as soma
+import yaml
+from cellxgene_census.experimental.pp import highly_variable_genes
+
+import scvi
+
+file = "scvi-config.yaml"
+
+with open(file) as f:
+    config = yaml.safe_load(f)
+
+census = cellxgene_census.open_soma()
+
+experiment_name = "mus_musculus"
```

Comment:
We are going to need to make this script parameters and or loop for both organisms

Reply:
Do we need one model for each experiment?

Reply:
Yes